councilmatic_core/management/commands/import_shapes.py

```diff
@@ -0,0 +1,38 @@
+import json
+
+from django.core.management.base import BaseCommand
+from councilmatic_core import models
+
+
+class Command(BaseCommand):
+    help = "Import boundary shapefiles for Post entities"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'shape_file',
+            help=(
+                'The location of the file containing shapes for each Division, '
+                'relative to the project root. The file should be formatted as JSON '
```

Comment:
Why not formatted as GeoJSON?

Reply:
From an authoring perspective, I think it would be easier to get or make a geojson file and then edit the properties to add division ids, so the geojson file would look something like:

```python
{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "geometry": {
        "type": "Point",
        "coordinates": [102.0, 0.5]
      },
      "properties": {
        "division_id": "/some/intersting/unique_id/division_id"
      }
    },
...
```

otherwise, we'll need to get some shapefile and transform it into this special format geojson file?
```